### PR TITLE
testing pr view in consult-gh

### PR DESCRIPTION
**Pull request by armindarvish**
This is a test PR to see how consult-gh-pr-view works!

### CODE REVIEW

File: another-file.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+plt.style.use("bosch")
+x = np.linspace(0, 2 * np.pi, 100)
+y = np.sin(x)
```
Comment (armindarvish) [inline another-file.py:6]: test comment

**Review by armindarvish — COMMENTED** [review]
i just reviewed the code

**Review by armindarvish — COMMENTED** [review]
test comment from gh cli

File: another-file.py (hunk 1 above)
Comment (armindarvish) [reply another-file.py:6]: A reply to a comment!

**Comment by armindarvish** [comment]
New comment!

File: another-file.py (hunk 1 above)
Comment (armindarvish) [inline another-file.py:6]: Anothe comment

File: another-file.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+plt.plot(x, y)
+
+x = np.linspace(0, 2 * np.pi, 1000)
+y = np.sin(x) + np.cos(10 * x) + np.random.random(len(x))
+plt.plot(x, y)
+
+plt.xlabel("time")
+plt.ylabel("data")
+plt.show()
```
Comment (armindarvish) [inline another-file.py:15]: Check this line out!

**Review by armindarvish — COMMENTED** [review]
Here is yet another test review!

**Comment by armindarvish** [comment]
# This is a test to see how the header shows up in consult-gh

```emacs-lisp
(print "Tesint Header")
```

*Bold* **Bold** _italic_ ~~strikethrough~~  

> This is quote!

**Comment by armindarvish** [comment]
this is a test comment

**Comment by armindarvish** [comment]
This is a test comment from cli to undertsand how to link to commits

**Review by armindarvish — COMMENTED** [review]
This is a test review

**Review by shivradnimra — COMMENTED** [review]
This is a test review

**Review by armindarvish — COMMENTED** [review]
test comment

**Review by armindarvish — COMMENTED** [review]
interesting

File: add_new_file.org [hunk 3]
```diff
@@ -0,0 +1,8 @@
+This is a new file
+
+#+begin_src emacs-lisp
+(print "new file!")
```
Comment (armindarvish) [inline add_new_file.org:4]: add more text here please

**Review by armindarvish — COMMENTED** [review]
This is a test review with comment on a patch

File: add_new_file.org [hunk 4]
```diff
@@ -0,0 +1,8 @@
+This is a new file
+
```
Comment (armindarvish) [inline add_new_file.org:2]: add more text here please

**Review by armindarvish — COMMENTED** [review]
This is a test review with comment on a patch

File: add_new_file.org [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+#+begin_src emacs-lisp
+(print "new file!")
+#+end_src
+
+#+RESULTS:
+: new file!
```
Comment (armindarvish) [inline add_new_file.org:8]: add more text here please

**Review by armindarvish — COMMENTED** [review]
This is a test review with comment on a patch

File: add_new_file.org [hunk 6]
```diff
@@ -0,0 +1,8 @@
+This is a new file
+
+#+begin_src emacs-lisp
+(print "new file!")
+#+end_src
+
+#+RESULTS:
```
Comment (armindarvish) [inline add_new_file.org:7]: add some text here please

File: add_new_file.org (hunk 3 above)
Comment (armindarvish) [inline add_new_file.org:4]: add more text here please

**Review by armindarvish — COMMENTED** [review]
This is a test review with comment on a patch

File: add_new_file.org (hunk 3 above)
Comment (armindarvish) [inline add_new_file.org:4]: add more text here please

**Review by armindarvish — COMMENTED** [review]
This is done with gh api

File: add_new_file.org (hunk 3 above)
Comment (armindarvish) [inline add_new_file.org:4]: add more text here please

**Review by armindarvish — COMMENTED** [review]
This is done with gh api

**Pull request merged**